src/components/TaskBody.vue

```diff
@@ -26,7 +26,7 @@ License along with this library.  If not, see <http://www.gnu.org/licenses/>.
 		:task-id="task.uri"
 		:class="{
 			'task-item--closed': task.closed,
-			'task-item--deleted': !!deleteTimeout,
+			'task-item--deleted': !!task.deleteCountdown,
```

Comment:
Should be compared to `null`, otherwise it will break if deleteCountdown is `0`.